python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,7 +42,7 @@ def _getName(obj):
 
 _imported_GangaList = None
 
-do_not_copy = ['_index_cache', '_parent', '_registry', '_data', '_read_lock', '_write_lock', '_proxyObject']
+do_not_copy = ['_index_cache', '_index_cache_dict', '_parent', '_registry', '_data', '_data_dict', '_read_lock', '_write_lock', '_proxyObject']
```

Comment:
as far as I can tell ```do_not_copy``` will need to exist for both a) the proxy layer and b) ```_parent``` not to get into infinite copy loops. As for the object vs dictionary does deep copy actually see setters and such or just the content of ```__dict__```?